lib/location/location.c

```diff
@@ -278,6 +284,9 @@ const char *location_method_str(enum location_method method)
 		return LOCATION_METHOD_WIFI_STR;
 
 	default:
+		if (method == LOCATION_METHOD_INTERNAL_WIFI_CELLULAR) {
+			return LOCATION_METHOD_WIFI_CELLULAR_STR;
+		}
```

Comment:
Why is this in the `default` branch instead of own case for `LOCATION_METHOD_INTERNAL_WIFI_CELLULAR`?

Reply:
It's not part of the enum so compiler is complaining about it:
```
/home/tor1/work/ncs/nrf/lib/location/location.c:285:9: warning: case value '100' not in enumerated type 'enum location_method' [-Wswitch]
  285 |         case LOCATION_METHOD_INTERNAL_WIFI_CELLULAR:
```

Reply:
Ok, makes sense, just looks funny. Maybe you could add a comment why it's like this.